Dockerfile

```diff
@@ -291,7 +291,7 @@ RUN pip install --upgrade cython && \
     pip install annoy && \
     # Need to use CountEncoder from category_encoders before it's officially released
     pip install git+https://github.com/scikit-learn-contrib/categorical-encoding.git && \
-    pip install google-cloud-automl && \
+    pip install google-cloud-automl==1.0.1 && \
```

Comment:
Could you add a comment explaining why we are pinning the version?

Reply:
Sure, done.